packages/share_plus/share_plus/example/lib/main.dart

```diff
@@ -25,9 +25,9 @@ class DemoAppState extends State<DemoApp> {
   String text = '';
   String subject = '';
   List<String> imagePaths = [];
-  String? filePath;
+  String filePath = '';
 
-  ShareWithAppWindows appSelected=ShareWithAppWindows.BY_DEFAULT_APP;
+  ShareWithApp appSelected = ShareWithApp.BY_DEFAULT_APP;
```

Comment:
there's a compilation error with the example, I guess it is because of the federated plugin structure, but we would need to double check if it is the case or not

Reply:
I checked no such error at my end.

Reply:
Resolved and Example app is running.
Commit 7ce1b3b44f30660e8077e7b3bed9d8f0f246ebe1